.cloudbuild/samples_build.yaml

```diff
@@ -0,0 +1,32 @@
+steps:
+- name: gcr.io/cloud-devrel-public-resources/java8
+  entrypoint: ls
+  args: [
+    '-alt',
+  ]
+- name: gcr.io/cloud-devrel-public-resources/java8
+  entrypoint: curl
+  args: [
+    '--header',
+    'Metadata-Flavor: Google',
+    'http://metadata.google.internal/computeMetadata/v1/instance/service-accounts/default/email'
+  ]
+- name: gcr.io/cloud-devrel-public-resources/java8
+  entrypoint: pwd
+- name: gcr.io/cloud-devrel-public-resources/java8
+  entrypoint: bash
+  args: [
+    '.kokoro/build.sh'
+  ]
+  env:
+  - 'JOB_TYPE=samples'
+  - 'GOOGLE_CLOUD_PROJECT=cloud-java-ci-sample'
+  - 'BIGTABLE_TESTING_INSTANCE=instance'
```

Comment:
not relevant to this repo - can we put this general version in synthtool templates instead if no custom modifications are needed?

Reply:
Custom modifications are needed for multiple repositories. I believe maintaining this file without templating shouldn't cause pains. Do you expect otherwise?

Reply:
Up to you, I don't have a strong preference if you think it's easier to maintain!

Reply:
I keep it not templaed then.

I removed `'BIGTABLE_TESTING_INSTANCE=instance'`